common/server_protocolhandler.cpp

```diff
@@ -171,6 +176,9 @@ Response::ResponseCode Server_ProtocolHandler::processRoomCommandContainer(const
     if (!room)
         return Response::RespNotInRoom;
 
+    lastActionReceived = timeRunning;
+    idleClientWarningSent = false;
```

Comment:
These two lines of code are repeated a lot. It would be better to hide them under a `resetIdleTimer()` function